mobile/src/features/Scan/common/useTriggerScanAction.tsx

```
@@ -57,6 +57,11 @@ export const useTriggerScanAction = ({
       const claimErrorDialog = claimErrorResolver(error)
       if (claimErrorDialog) {
         Alert.alert(claimErrorDialog.title, claimErrorDialog.message)
+      } else {
+        Alert.alert(
+          'Error',
+          'An unexpected error occurred during claim processing',
+        )
```

it was on purpose, in case of unknown error we failed silently.